### DISCUSSION TITLE
New Menu Style? "Basic" from Arctic Zephyr 2 Resurrection?

### DISCUSSION THREAD

**Discussion by inihendrix** [jurialmunkey/skin.arctic.horizon.2#873] [Discussion]
Hey @jurialmunkey just curious, how difficult would it be to add the 'Basic' menu style from Arctic Zephyr 2 Resurrection? For context, it appears as a bar of options on the bottom of the screen.

![Screenshot_2023-03-15-17-50-05-04_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/73711632/225481608-458ccca4-2d42-4020-82af-3a81d303962a.jpg)

Furthermore, this Basic menu style can be light or dark theme.

Would love to hear your thoughts!

Thank you!

**Comment by jurialmunkey** [comment]
It's not about difficulty. There were just too many things in AZ2 where I ended up making compromises and the skin became a bit of a mess. I don't have any intention of revisiting it. Plus it doesn't match the style of AH2 anyway.

There's already an option for a text based menu in AH2. If you combine it with standard fanart based fullscreen widgets then you can get a simple setup comparable to AZ2 basic menu.

**Comment by inihendrix** [comment]
Ah that's interesting, thank you for the insight!

As far as the main interest I had in that Basic menu style, it was the fact that the bar that appeared at the bottom of the screen allowed maximum unobstructed views of whatever content was playing at the time. And that the widgets in whatever tab were sort of hidden "under" the home menu bar. It required pressing down on the various tabs in order to access them.

Instead of having widgets always in full display, the home screen sort of resembled a windows desktop. I thought it was a great concept and always wondered why that particular menu style was only attempted for that skin.

Either way, I'm loving what you're doing in AH2 and just wanted to feed my curiosity haha.
